packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -5,6 +5,8 @@ const { getIntlConfig } = require('./configure');
 
 const debug = require('debug')('gasket:plugin:intl:buildModules');
 
+const rePkgParts = /^(?<name>(?:@[^/]+\/)?[\w-]+)(?<dir>\/.+)?/;
```

Comment:
Package names may have `_` as well; is that a word character? Maybe would be safer to do this:

```suggestion
const rePkgParts = /^(?<name>(?:@[^/]+\/)?[^/]+)(?<dir>\/.+)?/;
```